siliconcompiler/targets/asap7_demo.py

```diff
@@ -27,25 +27,20 @@ def setup(chip):
     #3. Select default flow/PDK
     chip.set('option', 'flow', 'asicflow')
     chip.set('option', 'pdk', 'asap7')
+    chip.set('option', 'stackup', '10M')
```

Comment:
Is `10M` right for asap7, or is it possible we accidentally copied it from freepdk45?